pages/dkp/kaptain/2.0.0/configuration/index.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: layout.pug
+navigationTitle: Configuration
+title: Configuration
+menuWeight: 10
+excerpt: Configuration settings for Kaptain
+enterprise: false
+---
+[//]: # "WARNING: This page is auto-generated and should not be modified directly."
+
+This page lists Kaptain configuration properties that can be adjusted during the initial installation or by updating the existing instance of Kaptain. These parameters are typically set in `params.yaml`.
+To learn more about overriding default parameter values as well as updating parameter on the existing Kaptain instance, refer to the [Overriding parameters][kudo_docs] and the [Update Parameters on Instance][kudo_docs_2] pages in KUDO documentation.
```

Comment:
@gvasquezvargas

Reply:
this needs to be fixed here if we stick to the generating tooling: https://github.com/mesosphere/kaptain/blob/master/scripts/generate-configuration-docs.py